tests/Unit/HttpClientTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+
+namespace Spatie\LaravelRay\Tests\Unit;
+
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Http;
+use Spatie\LaravelRay\Tests\TestCase;
+
+class HttpClientTest extends TestCase
+{
+    protected $minimumLaravelVersion = '8.45.0';
```

Comment:
This can be remove, let's just hardcode the number in the setup method.

Reply:
👍